src/EnrollQuestionController.js

```diff
@@ -94,7 +94,9 @@ function (Okta, FormController, Footer, TextBox) {
       .then(function(questionsRes) {
         var questions = {};
         _.each(questionsRes, function (question) {
-          questions[question.question] = question.questionText;
+          var localizedQuestion = Okta.loc('security.' + question.question);
```

Comment:
Let's write a Util helper function to do this logic so that we don't have to duplicate it